webapp/src/components/contentBlock.tsx

```diff
@@ -94,6 +99,7 @@ const ContentBlock = (props: Props): JSX.Element => {
                                 id='insertAbove'
                                 name={intl.formatMessage({id: 'ContentBlock.insertAbove', defaultMessage: 'Insert above'})}
                                 icon={<AddIcon/>}
+                                position='top'
```

Comment:
@asaadmahmood @wuwinson this shows submenu like this:
<img width="498" alt="image" src="https://user-images.githubusercontent.com/5587620/197382812-2877eeec-934d-4bb4-ad82-bc3c7778cb35.png">
and helps to avoid increasing scroll height of the card dialog.

What do you think about such change?